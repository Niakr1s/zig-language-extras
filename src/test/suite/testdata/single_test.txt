1/1 test.add 2... expected 2, found 3
FAIL (TestExpectedEqual)
/Users/ianic/code/vscode/zig_extras/test_project/src/main.zig:29:5: 0x1009406e3 in test.add 2 (test)
    try testing.expectEqual(2, 2 + 3);
    ^
0 passed; 0 skipped; 1 failed.
error: the following test command failed with exit code 1:
/Users/ianic/code/vscode/zig_extras/test_project/zig-cache/o/5e395467506bfb2f4e55a73f4998ccea/test