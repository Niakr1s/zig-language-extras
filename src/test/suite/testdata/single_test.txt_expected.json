[
  {
    "file": "/Users/ianic/code/vscode/zig_extras/test_project/src/main.zig",
    "line": 28,
    "column": 4,
    "message": "expected 2, found 3",
    "severity": 0
  }
]